package_manifest.yaml

```diff
@@ -137,18 +136,8 @@ foreman_packages:
       el9:
         - "el9-foreman-{{ foreman_version }}-staging"
     repoclosure_lookaside_repos:
-      el9:
-        - el9-baseos
-        - el9-appstream
-        - el9-puppet-7
-        - "el9-candlepin-{{ candlepin_version }}-staging"
-      el8:
-        - el8-baseos
-        - el8-appstream
-        - el8-extras
-        - el8-powertools
-        - el8-puppet-7
-        - "el8-candlepin-{{ candlepin_version }}-staging"
+      el8: "{{ hostvars['foreman-staging-repoclosure-el8']['repoclosure_lookaside_repos']['el8'] }}"
+      el9: "{{ hostvars['foreman-staging-repoclosure-el9']['repoclosure_lookaside_repos']['el9'] }}"
```

Comment:
I debated:
```yaml
repoclosure_lookaside_repos: "{{ hostvars['foreman-staging-repoclosure-{{ repoclosure_target_dist }}']['repoclosure_lookaside_repos'] }}"
```

But was worried that's too much magic.

Reply:
There is value in being explicit and here I appreciate it.